packages/frontend/editor-ui/src/stores/assistant.store.ts

```diff
@@ -116,9 +116,7 @@ export const useAssistantStore = defineStore(STORES.ASSISTANT, () => {
 		() => isAssistantEnabled.value && EDITABLE_CANVAS_VIEWS.includes(route.name as VIEWS),
 	);
 	const hideAssistantFloatingButton = computed(
-		() =>
-			(route.name === VIEWS.WORKFLOW || route.name === VIEWS.NEW_WORKFLOW) &&
-			!workflowsStore.activeNode(),
+		() => EDITABLE_CANVAS_VIEWS.includes(route.name as VIEWS) && !workflowsStore.activeNode(),
```

Comment:
This change fixes multiple "Ask AI" floating button shown in the debug route
<img width="169" height="739" alt="Image" src="https://github.com/user-attachments/assets/bfb53fad-5c26-4786-a7e7-71acecb2d837" />